protocol/packages/currencies/src/lease/mod.rs

```diff
@@ -1,14 +1,17 @@
 use serde::{Deserialize, Serialize};
 
-use sdk::schemars::{self, JsonSchema};
-
 use currency::{AnyVisitor, Group, Matcher, MaybeAnyVisitResult, SymbolSlice};
+use sdk::schemars::{self, JsonSchema};
 
-#[cfg(dex = "astroport")]
+#[cfg(feature = "astroport")]
 pub(crate) mod astroport;
+#[cfg(feature = "astroport")]
+use self::astroport as impl_mod;
 
-#[cfg(all(not(dex = "astroport"), dex = "osmosis"))]
+#[cfg(any(feature = "osmosis", feature = "testing"))]
 pub(crate) mod osmosis;
+#[cfg(any(feature = "osmosis", feature = "testing"))]
+use self::osmosis as impl_mod;
```

Comment:
do not think we need `feature = "testing"`. The testing feature should just add `derive(Debug)`. The consumers of this package should ask for `--features osmosis, testing` or `--features astroport, testing`